types/mithril/test/test-api.ts

```diff
@@ -1,8 +1,8 @@
 // Typescript adaptation of mithril's test suite.
 // Not intended to be run; only to compile & check types.
 
-import * as m from 'mithril';
-import * as stream from 'mithril/stream';
+import m = require('mithril');
+import stream = require('mithril/stream');
```

Comment:
@spacejack We really *should* convert Mithril's value-level exports to *just* be default exports, so we remain ES6-compatible. Yes, it's breaking, but we're going to have to do it regardless.